noxfile.py

```diff
@@ -29,6 +28,7 @@
     "tests": "tests/requirements.txt",
     "common-wheels": "tests/requirements-common_wheels.txt",
 }
+HERE = Path(__file__).parent
```

Comment:
How about a descriptive name?
```suggestion
GIT_REPOSITORY_ROOT = Path(__file__).parent
```